EventRouter/log/logger.go

```diff
@@ -8,8 +8,8 @@ import (
 type Level int8
 
```

Comment:
EventRouter/log package could be moved to a common module? (That is if we are going to have multiple modules in the end.)